src/Css/Specificity/Specificity.php

```diff
@@ -47,27 +38,48 @@ public function __construct($a = 0, $b = 0, $c = 0)
     }
 
     /**
-     * Increase the current specificity by adding the three values
+     * @param Specificity $specificity
+     *
+     * @return Specificity
+     */
+    public function plus(Specificity $specificity)
+    {
+        return new self($this->a + $specificity->a, $this->b + $specificity->b, $this->c + $specificity->c);
+    }
+
+    /**
+     * Returns global specificity value.
      *
-     * @param int $a The number of ID selectors in the selector
-     * @param int $b The number of class selectors, attributes selectors, and pseudo-classes in the selector
-     * @param int $c The number of type selectors and pseudo-elements in the selector
+     * @return int
      */
-    public function increase($a, $b, $c)
+    public function getValue()
```

Comment:
I refactores it to look the same as https://github.com/symfony/css-selector/blob/master/Node/Specificity.php, but the code should be refactored to use the Symfony version in a near future.